code/game/gamemodes/cult/cult_items.dm

```diff
@@ -96,6 +99,7 @@
 	icon_state = "magusred"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|FEET|ARMS|HANDS
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
+	origin_tech = list(TECH_ARCANE = 3)

```

Comment:
As with the sword, this can be found in xenoarch commonly enough that it's best to stay at the base 2.